README.adoc

```diff
@@ -245,40 +245,39 @@ Create the `FaultToleranceTest` class in the
 include::finish/src/test/java/it/io/openliberty/guides/faulttolerance/FaultToleranceTest.java[tags=ft_testing;!javadoc]
 ----
 
-The `@BeforeClass` annotation is placed on a method that executes before any of the test cases. In
+The `@BeforeClass` annotation is placed on a method that executes before any of the other test cases. In
 this case, the `oneTimeSetup()` method retrieves the port number for the Open Liberty server and 
 builds a base URL string that is used throughout the tests.
 
-The `@Before` and `@After` annotations are placed on methods that execute before and after every 
-test case. These methods are generally used to perform any setup and teardown tasks. In this case,
-the setup method creates a JAX-RS client which makes HTTP requests to the `inventory` service. This
-client must also be registered with a JSON-P provider (JsrJsonpProvider) to process JSON resources. 
-The teardown method simply destroys this client instance as well as the HTTP responses, resets the
+The `@Before` and `@After` annotations are placed on methods that execute before and after any 
+other test case. These methods are generally used to perform any setup and teardown tasks. In this case,
+the setup method creates a JAX-RS client, which makes HTTP requests to the `inventory` service. This
+client must also be registered with a `JsrJsonpProvider` JSON-P provider to process JSON resources. 
```

Comment:
We don't need "`JsrJsonpProvider`". I think it was put in the () to spell out the JSON-P acronym.

Reply:
OK, we can delete "JsrJsonpProvider."